docs/notebook/kuberay-on-kind.ipynb

```diff
@@ -140,8 +140,7 @@
     }
    ],
    "source": [
-    "!kubectl create -k \"github.com/ray-project/kuberay/manifests/cluster-scope-resources\"\n",
-    "!kubectl apply -k \"github.com/ray-project/kuberay/manifests/base\""
+    "kubectl create -k \"github.com/ray-project/kuberay/ray-operator/config/default?ref=v0.3.0&timeout=90s\""
```

Comment:
This command needs to be prefixed with a `!` to be executable in a Jupyter Notebook.